src/locales/en-GB.json

```diff
@@ -115,7 +115,7 @@
   "$clickToZoomEnabled": "Mouse Click To Zoom",
   "$reducedMotion": "Reduce motion (disables animations)",
   "$preserveViewport": "Preserve Zoom",
-  "$uvWebsite": "<a href='https://github.com/universalviewer/universalviewer'>more info</a>",
+  "$uvWebsite": "<a href='https://github.com/universalviewer/universalviewer'>more info about the Universal Viewer</a>",
```

Comment:
I suggest we capitalise the first letter to bring the formatting inline with other UV text ('More...')

Reply:
No objection here!

Reply:
Agreed! Happy to adjust along with updating the translations